### DISCUSSION TITLE
启用任意无障碍工具后chrome搜索框出现异常行为

### DISCUSSION THREAD

**Discussion by hnpp2** [gkd-kit/gkd#839] [Q&A] [answered]
打开gkd无障碍服务，会导致点击chrome主页的搜索栏时，出现输入法连续跳动两次的问题。我把规则全部关闭也无法解决这个问题，只要打开无障碍服务就能复现。虽然不是啥恶性问题，但是看起来很不舒服。之前没搞清楚状况，我也因此重置了一次手机。今天才发现有这样的问题。
![1735094805039](https://github.com/user-attachments/assets/532c31dc-b0d3-4920-a621-0d552440636c)

**Comment by lisonge** [comment]
这不是 gkd 的问题

你关闭gkd后打开其它无障碍软件也会出现相同的问题

**Reply by hnpp2** [reply]
谢谢您，那请问大概是个什么原理？有没有办法解决？

**Reply by lisonge** [reply]
测试了一下，只是开启无障碍，不执行任何操作，甚至将 chrome 手动配置在 xml 列表之外，还是会出现这个上面的异常行为

应该是 chrome 的问题

**Reply by lisonge** [reply]
你可以在进入 chrome 时暂时手动关闭 gkd

https://gkd.li/guide/external#shortcut

**Reply by hnpp2** [reply]
好的谢谢您，那我去chrome那边反应一下bug，我没啥问题了，请问我这条评论是否需要删除？

**Reply by lisonge** [reply]
不用删除

---

刚刚测试了一下基于 shizuku+UiAutomation 的无障碍匹配不会出现这个问题(_因为没有系统无障碍服务在运行_)，同时也能防止目标应用(如银行相关应用)检测无障碍服务，后续会支持这个模式

**Reply by iddsjo** [reply]
> 好的谢谢您，那我去chrome那边反应一下bug，我没啥问题了，请问我这条评论是否需要删除？

Chrome还是没修这个bug

**Reply by lisonge** [reply]
使用最新测试版

**Reply by iddsjo** [reply]
> 使用最新测试版

最新的Chrome dev,beta,canary都一样，没修复

**Reply by plane714** [reply]
> 使用最新测试版

现在好像 chrome 还是会这样，请问有什么解决办法吗

**Reply by lisonge** [reply, marked as answer]
使用自动化模式，或者将 Chrome 加到局部关闭的名单

**Comment by chaoliu615** [comment]
感谢找到问题了，本来我还以为是系统问题，或者浏览器的问题。没想到症结在无障碍

**Reply by iddsjo** [reply]
新版Chrome已经修了，但有新的bug，第一次点搜索框输入法会消失一次

**Reply by plane714** [reply]
> 新版Chrome已经修了，但有新的bug，第一次点搜索框输入法会消失一次

正式版没修 依旧是弹出然后收起的 bug
